shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt

```diff
@@ -29,7 +29,7 @@ import com.karumi.shot.compose.ComposeScreenshotRunner
 import com.karumi.shot.compose.ScreenshotMetadata
 import java.lang.IllegalStateException
 
-interface ScreenshotTest {
+interface ScreenshotTest : FocusCleaner {
 
     private val context: Context get() = getInstrumentation().targetContext
```

Comment:
Two remarks:
First, I would gladly write a Test for this, but I had problems with shot-consumer. I can see it under project view in Android Studio (but not under Android view, as I can see :core or :shot), but when executing ./gradlew shot-consumer:executeScreenshotTest I get:
* What went wrong:
Project 'shot-consumer' not found in root project 'Shot'.

If you could help me with that, I would add a test for this

Secondly, it could be also interesting to implement something similar for Jetpack Compose, but I guess we will still need to wait for its release and see what are Compose needs regarding Snapshot testing

Reply:
These projects are not connected to the root folder. You need to open them specifically with Android Studio.